content/en/network_monitoring/network_path/setup.md

```diff
@@ -210,6 +210,50 @@ Agent `v7.61+` is required.
 
 For network traffic paths on Windows environments, only detected TCP connections are shown.
 
+**Note**: Network traffic paths is experimental and is not yet stable. Do not deploy network traffic paths widely in a production environment.
```

Comment:
I think even in doc, we should avoid duplicating if possible, to ease maintenance.